astropy/convolution/convolve.py

```diff
@@ -289,13 +289,15 @@ def convolve(
 
     # Check dimensionality
     if array_internal.ndim == 0:
-        raise Exception("cannot convolve 0-dimensional arrays")
+        raise ValueError("cannot convolve 0-dimensional arrays")
```

Comment:
Does change the exception type, so maybe better not backport.

Other than that, since I haven't heard from Tom R but there is an approval, I am just going to merge.

Thanks!